### PR TITLE
Add the article When Good Enough Software is Best

### PR DESCRIPTION
**Pull request by marcelotrevisani**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 * [Python Culture](https://svn.python.org/projects/sandbox/trunk/dev/culture.rst) - python.org historic document
 * [TechnicalDebt](https://martinfowler.com/bliki/TechnicalDebt.html) - Martin Fowler
 * [Top 7 programmers bad habits](https://web.archive.org/web/20110803100310/http://www.makinggoodsoftware.com/2011/05/23/top-7-programmers-bad-habits) - Alberto Gutierrez
+* [When Good Enough Software is Best](https://ieeexplore.ieee.org/stamp/stamp.jsp?arnumber=382191) - Edward Yourdon
```
Comment (nicoddemus) [inline README.md:36]: Does this need to be paid? 🤔

Comment (marcelotrevisani) [reply README.md:36]: I didn't check it sorry.
I was using my account from UFMG which I have access to the IEEE
:smile:

Comment (marcelotrevisani) [reply README.md:36]: I will close it then :)

Comment (nicoddemus) [reply README.md:36]: Thanks anyway for the intention! 😁

Comment (igortg) [reply README.md:36]: Found this link on Google Scholar: https://www.liamwho.com/wp-content/uploads/When-Good-Enough-Software-Is-Best.pdf

@marcelotrevisani, can you confirm if this is the same paper? Can you reopen the PR if this is the case?

Comment (marcelotrevisani) [reply README.md:36]: Yeap! It is the same article! :)

Comment (marcelotrevisani) [reply README.md:36]: I think I deleted this fork :/
I will create another PR for that then
Thanks @igortg  !

**Pull request closed without merging**